caddy/frankenphp/Caddyfile

```diff
@@ -43,3 +45,9 @@
 
 	php_server
 }
+
+# As an alternative to editing the above site block, you can add your own site
+# block files in the Caddyfile.d directory, and they will be included as long
+# as they use the .caddyfile extension.
+
+import Caddyfile.d/*.caddyfile
```

Comment:
WDYT @mholt @francislavoie, is this a good Caddy practice?

Reply:
This comes from the caddy packages, so I would hope so.

I've adopted it since and it has been a good decision. Separation of concerns, which is also why I'm working on https://github.com/dunglas/frankenphp/issues/1490.